src/WorkItemMigrator/Migration.Common/Journal.cs

```diff
@@ -29,21 +29,21 @@ internal static Journal Load(Journal journal)
         {
             if (File.Exists(journal.ItemsPath))
             {
-                var revLines = File.ReadAllLines(journal.ItemsPath);
-                foreach (var rev in revLines)
+                string[] revLines = File.ReadAllLines(journal.ItemsPath);
+                foreach (string rev in revLines)
                 {
-                    var props = rev.Split(';');
+                    string[] props = rev.Split(';');
                     journal.ProcessedRevisions[props[0]] = (Convert.ToInt32(props[1]), Convert.ToInt32(props[2]));
                 }
             }
 
             if (File.Exists(journal.AttachmentsPath))
             {
-                var attLines = File.ReadAllLines(journal.AttachmentsPath);
-                foreach (var att in attLines)
+                string[] attLines = File.ReadAllLines(journal.AttachmentsPath);
+                foreach (string att in attLines)
                 {
-                    var props = att.Split(';');
-                    journal.ProcessedAttachments[props[0]] = Convert.ToInt32(props[1]);
+                    string[] props = att.Split(';');
+                    journal.ProcessedAttachments[props[0]] = props[1];
```

Comment:
The old code uses the **Attachment** type from WorkItemTracking.Client, to create an attachment offline, and to then upload it when we call WorkItem.Save(), if I'm not mistaken.

Attachment had an integer id, but since I refactored away WorkItemTracking.Client and began using WorkItemTracking.WebApi instead, I needed some other way to identify the attachments, so I went with the GUID instead.